doc/btp_aics.txt

```diff
@@ -5,9 +5,9 @@ Commands and responses:
 
         Opcode 0x00 - Error response
 
-        Opcode 0x01 - Read supported commands
+        Opcode 0x01 - Read Supported Commands command/response
 
-                Controller Index:       <controller id>
```

Comment:
A general comment: While I'm not 100% sure how the `controller id` is used in AutoPTS, if you can send commands to different controllers, then there's a decent chance that each of those have different supported commands, right? 

Can you elaborate on why it's being changed to `non-controller`?